dataeng/jobs/analytics/SnowflakeMicrobachelorsITK.groovy

```diff
@@ -17,7 +16,10 @@ class SnowflakeMicrobachelorsITK {
 
     public static def job = { dslFactory, allVars ->
         dslFactory.job('snowflake-microbachelors-send-coaching-data-itk') {
-            logRotator common_logrotator
+            logRotator {
+                daysToKeep(30)
+                artifactDaysToKeep(5)
```

Comment:
Suggestion: keep 7 days for sanity / time zone issues.

Reply:
Sure!